src/mscorlib/shared/System/MissingFieldException.cs

```diff
@@ -9,64 +9,50 @@
 **
 =============================================================================*/
 
-
-using System;
-using System.Runtime.Remoting;
 using System.Runtime.Serialization;
-using System.Runtime.CompilerServices;
-using System.Globalization;
 
 namespace System
 {
     [Serializable]
-    public class MissingFieldException : MissingMemberException, ISerializable
+    public class MissingFieldException : MissingMemberException
     {
         public MissingFieldException()
             : base(SR.Arg_MissingFieldException)
         {
             HResult = __HResults.COR_E_MISSINGFIELD;
         }
 
-        public MissingFieldException(String message)
+        public MissingFieldException(string message)
             : base(message)
         {
             HResult = __HResults.COR_E_MISSINGFIELD;
         }
 
-        public MissingFieldException(String message, Exception inner)
+        public MissingFieldException(string message, Exception inner)
             : base(message, inner)
         {
             HResult = __HResults.COR_E_MISSINGFIELD;
         }
 
-        protected MissingFieldException(SerializationInfo info, StreamingContext context) : base(info, context)
+        public MissingFieldException(string className, string methodName)
         {
+            ClassName = className;
+            MemberName = methodName;
         }
 
-        public override String Message
+        protected MissingFieldException(SerializationInfo info, StreamingContext context)
+            : base(info, context)
         {
-            get
-            {
-                if (ClassName == null)
-                {
-                    return base.Message;
-                }
-                else
-                {
-                    // do any desired fixups to classname here.
-                    return SR.Format(SR.MissingField_Name, (Signature != null ? FormatSignature(Signature) + " " : "") + ClassName + "." + MemberName);
```

Comment:
Hmm, this is implemented differently between CoreCLR and CoreRT. Not sure which side is right. I will exclude this file from this batch.